### PR TITLE
Instructions on handing interactive ISO Install and where to locate grub.cfg

### PR DESCRIPTION
**Pull request by LucasSaintarbor**
**Summary**
In reference to #3590 (https://github.com/harvester/harvester/issues/3590)...there may be times where someone may be dropped into grub2 while the USB based ISO is hooked up. 
- Added what partition to set as root & where to located the grub.cfg from the iso.

**Next steps**
- Is this information accurate? 
- Can any details or context be added?  
- Is there a better spot on the page for this info? 
- Add this info to v1.1

### CODE REVIEW

**Review by irishgordo — APPROVED** [review]
This looks good!
I'm adding @bk201 and @masteryyh  to review this as well just to double-check to make sure this is what we would want the docs to say.

File: docs/install/usb-install.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ Make sure you choose the correct device. The process erases data on the selected
 # sudo dd if=<path_to_iso> of=<path_to_usb_device> bs=64k
 ```
 
+Sometimes using a USB to mount the Harvester ISO disk directs you to Grub2. In this case, it helps to know what to set the root partition to and the location of `grub.cfg`.
+
+You can see the current root partition in the `grub.cfg`. To view the `grub.cfg` from the ISO, run the command `chainloader /boot/grub2/x86_64-efi/grub.efi`.
+
+To set a new root partition, run the command `set root = (hdN,msdos1)`, where N denotes N numbers of possible drives. N should be the drive your USB is mounted to.
```
Comment (irishgordo) [inline docs/install/usb-install.md:35]: the command should read:
`set root=(hdN,msdos1)` 

as if it has " " (spaces) in the command, the command will fail if I remember correctly :thinking: :smile:

**Review by irishgordo — CHANGES_REQUESTED** [review]

File: docs/install/usb-install.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ Make sure you choose the correct device. The process erases data on the selected
 # sudo dd if=<path_to_iso> of=<path_to_usb_device> bs=64k
 ```
 
+Sometimes using a USB to mount the Harvester ISO disk directs you to Grub2. In this case, it helps to know what to set the root partition to and the location of `grub.cfg`.
```
Comment (masteryyh) [inline docs/install/usb-install.md:31]: I think this should move to `Common issues`?

File: docs/install/usb-install.md [hunk 3]
```diff
@@ -30,6 +30,14 @@ Make sure you choose the correct device. The process erases data on the selected
...
 ## Common issues
 
+###
+
+Sometimes using a USB to mount the Harvester ISO disk directs you to Grub2. In this case, it helps to know how to set the root partition and where to locate `grub.cfg`.
+
+You can see the current root partition in the `grub.cfg`. To view the `grub.cfg` from the ISO, run the command `chainloader /boot/grub2/x86_64-efi/grub.efi`.
+
+To set a new root partition, run the command `set root=(hdN,msdos1)`, where N denotes N numbers of possible drives. N should be the drive your USB is mounted to.
+
```
Comment (LucasSaintarbor) [inline docs/install/usb-install.md:40]: @irishgordo In [issue#3590](https://github.com/harvester/harvester/issues/3590), you mentioned "Then they'd be able to fire off the boot command from Grub2 and be taken to the interactive ISO installation." Should that step be added here as well?

File: docs/install/usb-install.md [hunk 4]
```diff
@@ -30,6 +30,14 @@ Make sure you choose the correct device. The process erases data on the selected
 
 ## Common issues
 
+###
+
+Sometimes using a USB to mount the Harvester ISO disk directs you to Grub2. In this case, it helps to know how to set the root partition and where to locate `grub.cfg`.
+
+You can see the current root partition in the `grub.cfg`. To view the `grub.cfg` from the ISO, run the command `chainloader /boot/grub2/x86_64-efi/grub.efi`.
```
Comment (irishgordo) [inline docs/install/usb-install.md:37]:
```suggestion
You can see the current root partition in the `grub.cfg`, once you've set the root to be within where the ISO lives on the USB. To view the `grub.cfg` from the ISO, run the command `cat /boot/grub2/grub.cfg`.
```

So to view it, the user will need to look at the `grub.cfg` with something like `cat` - sadly at least on our Grub 2.06, it seems `less` isn't an available command :sweat_smile:  - but! - `cat` should work great for viewing.

File: docs/install/usb-install.md (hunk 3 above)
Comment (irishgordo) [inline docs/install/usb-install.md:40]:
```suggestion

To launch into the GNU GRUB2 Boot Menu from the ISO, once the root has been set to where the ISO is located at the partition of `msdos1` - you can run `chainloader /boot/grub2/x86_64-efi/grub.efi` to get loaded into the GRUB2 Boot Menu for the Harvester Installer.
```

Comment (irishgordo) [reply docs/install/usb-install.md:40]: Ahh I see what you're saying :smile:  - I've added a suggestion below :+1:

**Review by irishgordo — CHANGES_REQUESTED** [review]

File: docs/install/usb-install.md [hunk 5]
```diff
@@ -34,6 +34,15 @@ Make sure you choose the correct device. The process erases data on the selected
 
 ## Common issues
 
+###
+
+Sometimes using a USB to mount the Harvester ISO disk directs you to Grub2. In this case, it helps to know how to set the root partition and where to locate `grub.cfg`.
```
[the remark this answers is not in the document]
Comment (masteryyh) [reply docs/install/usb-install.md:39]: `directs you to GRUB2` is a bit confusing, if the problem is dropping into GRUB2 shell directly when booting up maybe we can describe it more clearly? (if I'm understanding this correctly)

**Review by masteryyh — APPROVED** [review]
LGTM, just a few suggestions ;)

**Comment by irishgordo** [comment]
I'm wondering if this is still relevant now that:
https://github.com/harvester/harvester/issues/3866

Has been fixed.
Should these instructions still be added?
As this all seems to be related, so users shouldn't encounter this in newer versions of Harvester.

cc: @masteryyh @bk201

**Comment by guangbochen** [comment]
Maybe consider adding this to the v1.1 doc only as v1.2 has already fixed it.

**Comment by bk201** [comment]
@LucasSaintarbor We need to rebase the PR to merge it, thanks!

**Comment by bk201** [comment]
@jillian-maroket Can you help rebase this PR?

File: docs/install/usb-install.md [hunk 6]
```diff
@@ -56,7 +56,16 @@ You can use the 'dd' command on Linux or other platforms to create a USB install
 
 ## Known issues
 
-### A `GRUB _` text is displayed, but nothing happens when booting from a USB installation flash drive
+###
+
+Sometimes using a USB to mount the Harvester ISO disk directs you to GRUB2. In this case, it helps to know how to set the root partition and where to locate `grub.cfg`.
```
Comment (LucasSaintarbor) [inline docs/install/usb-install.md:61]:
```suggestion
Sometimes when using a USB to mount the Harvester ISO disk, it drops you in the GRUB2 shell while booting. In this case, it helps to know how to locate `grub.cfg`  and set the root partition to proceed with the installation.
```

File: docs/install/usb-install.md [hunk 7]
```diff
@@ -56,7 +56,16 @@ You can use the 'dd' command on Linux or other platforms to create a USB install
...
+
+You can see the current root partition in the `grub.cfg`, once you've set the root to be within where the ISO lives on the USB. To view the `grub.cfg` from the ISO, run the command `cat /boot/grub2/grub.cfg`.
```
Comment (LucasSaintarbor) [inline docs/install/usb-install.md:63]:
```suggestion
To view the `grub.cfg` from the ISO, run the command `cat /boot/grub2/grub.cfg`. You can see the current root partition in the `grub.cfg`.
```

File: docs/install/usb-install.md [hunk 8]
```diff
@@ -56,7 +56,16 @@ You can use the 'dd' command on Linux or other platforms to create a USB install
...
+
+To set a new root partition, run the command `set root=(hdN,msdos1)`, where N denotes N numbers of possible drives. N should be the drive your USB is mounted to.
+
+To launch into the GNU GRUB2 Boot Menu from the ISO, once the root has been set to where the ISO is located at the partition of `msdos1` - you can run `chainloader /boot/grub2/x86_64-efi/grub.efi` to get loaded into the GRUB2 Boot Menu for the Harvester Installer.
```
Comment (LucasSaintarbor) [inline docs/install/usb-install.md:67]:
```suggestion
Once the root partition has been set, where the ISO is located in the `msdos1` partition, run the command `chainloader /boot/grub2/x86_64-efi/grub.efi` to get dropped into the GRUB2 Boot Menu for the Harvester Installer.
```

**Review by LucasSaintarbor — COMMENTED** [review]
@bk201 Sorry I missed this. Rebased this PR. 

Are these changes okay? If they are, I'll add them to the v1.1 docs per comment (https://github.com/harvester/docs/pull/298#issuecomment-1784696638)

**Review by bk201 — APPROVED** [review]
LGTM! Thanks!

**Comment by bk201** [comment]
Replace by https://github.com/harvester/docs/pull/478

**Pull request closed without merging**